test/e2e/page-objects/pages/onboarding/start-onboarding-page.ts

```diff
@@ -85,8 +81,9 @@ class StartOnboardingPage {
 
   async checkLoginPageIsLoaded(): Promise<void> {
     try {
+      // Wait for the page to load
+      await this.driver.delay(largeDelayMs);
```

Comment:
could we avoid using a hardcoded delay here?
Below we wait for multiple selectors to appear. 
Has the app become slower with these changes, which cause that the below elements take longer than 10 seconds to appear? 🤔